lib/rack/mock_cookie.rb

```diff
@@ -0,0 +1,26 @@
+module Rack
+  # Compatibility around CGI::Cookie
+  class MockCookie # :nodoc:
```

Comment:
I think it would be better to have this as `Rack::MockResponse::Cookie`, and move it to the `mock_response.rb` file, since `MockResponse` seems to be the only place it is used.

Reply:
Sure, that seems fine. I added private_constat too.